relay-general/src/pii/processor.rs

```diff
@@ -426,6 +428,36 @@ mod tests {
         assert_annotated_snapshot!(event);
     }
 
+    #[test]
+    fn test_span_default_rules() {
+        let ds_config = DataScrubbingConfig {
+            scrub_data: true,
+            scrub_defaults: true,
```

Comment:
Question: what fields are the default fields?

Reply:
sorry, I don't really understand the question

Reply:
What is the difference between `scrub_data` and `scrub_defaults`? What fields does `scrub_defaults` refer to?

Reply:
ah, so first is enabling the data scrubbing and the second is enabling default data scrubbers provided by relay, e.g. remove tokens, passwords, auth, ssh keys etc.